db/commit_error.go

```diff
@@ -0,0 +1,11 @@
+package db
+
+// CommitError is used to signal an error when trying to fast forward.
```

Comment:
FastForwardError woulda been more specific, but nbd

Reply:
I had that at first but thought it was to specific. It is internal only though.